cypress/page_object/task3/Header.js

```diff
@@ -0,0 +1,51 @@
+class Header {
+
+    findMenuItem(text, elNumber) {
```

Comment:
what is this for? why is is called like that? what are those parameters? why do you perform some checks inside?